### PR TITLE
Having cron do the backups routinely.

### PR DESCRIPTION
**Pull request by cggaurav**
@paalmoest Where do we set CRON_TIME?

### CODE REVIEW

**Comment by ahfeel** [comment]
+1 !

File: run.sh [hunk 1]
```diff
@@ -5,3 +5,8 @@ if [[ "$RESTORE" == "true" ]]; then
 else
   ./backup.sh
 fi
+
+echo "${CRON_TIME} /backup.sh >> /dockup.log 2>&1" > /crontab.conf
```
Comment (tifayuki) [inline run.sh:9]: Better to move them into a if clause: only execute when CRON_TIME is set

**Comment by kopax** [comment]
+1

**Comment by cggaurav** [comment]
Going for another PR from my fork instead.

**Pull request closed without merging**